README.md

```diff
@@ -0,0 +1,50 @@
+# Code Challenges
+A new collection of my personal solutions to various code challenges.
+### Setup
+This repo implements solutions in Python 3.8+.  It's recommended to run and install dependencies from a virtual environment.
+
+After cloning this repo:
+```sh
+python -m venv .venv
+```
+or if Python 3 is not your default.
+```sh
+python3 -m venv .venv
+```
+
+Then activate the virtual environment.
+```sh
+. ./.venv/bin/activate
+```
+Next install the dependencies:
+```sh
+pip install -r requirements.txt
+```
+### Testing
+Tests are setup to run with Pythons builtin unittest library.
+```sh
+python -m unittest
```

Comment:
Note I'm not yet using Pytest.  Might not.  Playing with Pythons built in testing module to see how it has improved.